scripts/create_php_tags.py

```diff
@@ -81,24 +77,25 @@ def parse_and_create_php_tags_file():
             # Also create a class tag when encountering a __construct()
             if tag_name == '__construct' and scope is not None:
                 scope, tag_name = split_scope(scope)
-                tag_list.append((tag_name, TYPE_CLASS, None, arg_list, scope))
+                tag_list.append((tag_name, TYPE_CLASS, None, arg_list, scope or ''))
 
     # write tags
     script_dir = dirname(__file__)
     tags_file_path = join(script_dir, '..', 'data', 'tags', 'std.php.tags')
-    with open(tags_file_path, 'w') as tags_file:
+    with open(tags_file_path, 'w', encoding='iso-8859-1') as tags_file:
         tags_file.write('# format=tagmanager\n')
         for tag_name, tag_type, return_type, arg_list, scope in sorted(tag_list):
-            tag_line = '{}'.format(tag_name)
-            for attr, type in [(tag_type, TA_TYPE),
+            tag_line = f'{tag_name}'
+            for attr, type_ in [(tag_type, TA_TYPE),
                                (arg_list, TA_ARGLIST),
                                (return_type, TA_VARTYPE),
                                (scope, TA_SCOPE)]:
                 if attr is not None:
-                    tag_line += '{type:c}{attr}'.format(type=type, attr=attr)
+                    tag_line += f'{type_:c}{attr}'
+                    print(tag_line)
```

Comment:
new debugging, is that wanted?

Reply:
Oops, thanks for spotting. Just removed it.